test/makecolor-wet-palette.test.mjs

```
@@ -0,0 +1,146 @@
+  const helperEnd = source.indexOf('// カラー作成制御オブジェクト');
+  const methodStart = source.indexOf('    _saveWetPaletteSnapshot() {');
+  const methodEnd = source.indexOf('    // 他システムが参照する色', methodStart);
+  assert.notEqual(helperStart, -1);
+  assert.ok(helperEnd > helperStart);
+  assert.notEqual(methodStart, -1);
+  assert.ok(methodEnd > methodStart);
+
+  const helperSource = source.slice(helperStart, helperEnd).replaceAll('export ', '');
+  const methodSource = source
+    .slice(methodStart, methodEnd)
+    .replace(/^    /gm, '')
```

ESLint

[error] Spaces are hard to count. Use {4}.

(no-regex-spaces)